### DISCUSSION TITLE
Release the Beast

### DISCUSSION THREAD

**Discussion by johnnyq** [nextcloud/all-in-one#984] [Questions] [answered]
Super excited about Nextcloud AIO 1.7 with the new Nextcloud 24.03
We usually do upgrades Friday nights was wondering if it was stable enough for release

**Comment by Zoey2936** [comment]
In my opinion yes, but I have very low requirements on stability. And my cloud is used by only a small amount of people.

**Comment by szaimen** [comment, marked as answer]
I will release 1.7.0 stable tomorrow if nothing comes up that blocks it

**Reply by szaimen** [reply]
done

**Comment by johnnyq** [comment]
Super!!!
